Sources/core/objects/OnMapBase.cpp

```diff
@@ -35,3 +36,8 @@ void IOnMapBase::AttackBy(id_ptr_on<Item> item)
 {
     // nothing
 }
+
```

Comment:
So this new function should not be needed - `Bump` will handle projectile events (it is better explained below).